### PR TITLE
LPS-184640

### PR DESCRIPTION
**Pull request by ces-locpham**
Hi team,
https://liferay.atlassian.net/browse/LPS-184640
Please help me review this ticket.

In this PR, I created a service wrapper for DLFolder. I chose to add to the document-library-service to place the service wrapper

Many thanks,
Loc
cc @tototrinh , @ces-quanhuynh , @ces-luannguyen

### CODE REVIEW

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/service/PermissionTemplateSettingDLFolderLocalServiceWrapper.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			long repositoryId, boolean mountPoint, long parentFolderId,
+			String name, String description, boolean hidden,
+			ServiceContext serviceContext)
+		throws PortalException {
+
+		DLFolder folder = super.addFolder(
+			externalReferenceCode, userId, groupId, repositoryId, mountPoint,
+			parentFolderId, name, description, hidden, serviceContext);
+
+		if (FeatureFlagManagerUtil.isEnabled("LPS-87806")) {
```
Comment (ces-quanhuynh) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/service/PermissionTemplateSettingDLFolderLocalServiceWrapper.java:39]: If you use Feature Flag here, please add the feature flag in portal.properties
see https://github.com/tototrinh/liferay-portal/commit/17744cdca783919d88d14a5945102920358fac67

**Comment by ces-locpham** [comment]
Hi @ces-quanhuynh ,
I updated it.

**Review by ces-quanhuynh — APPROVED** [review]

**Comment by ces-quanhuynh** [comment]
LGTM Thx!

**Review by ces-totrinh — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**